openquake/calculators/event_based_risk.py

```diff
@@ -46,6 +82,17 @@ def square(L, R, factory):
     return losses
 
 
+def old_loss_curves(asset_values, rcurves, ordinal, ratios):
+    """
+    Build loss curves in the old format (i.e. (losses, poes)) from
+    loss curves in the new format (i.e. poes).
+    """
```

Comment:
Uhm... instead, I will add an underscore in front of this function, after all it is an internal detail that may even disappear in the future.